.github/fixtures/test-keep-a-changelog-links-current-arg/cliff.toml

```diff
@@ -36,21 +35,35 @@ body = """
     {% endfor %}
 {% endfor %}\n
 """
-# remove the leading and trailing whitespace from the template
-trim = true
-# changelog footer
-footer = """
+# Whether to remove leading and trailing whitespaces from all lines of the changelog's body.
+trim_body_whitespace = true
+# A Tera template to be rendered as the changelog's footer (see https://keats.github.io/tera/docs/#introduction).
+footer_template = """
 <!-- generated by git-cliff -->
 """
 
-[git]
-# parse the commits based on https://www.conventionalcommits.org
-conventional_commits = true
-# filter out the commits that are not conventional
-filter_unconventional = true
-# process each line of a commit as an individual commit
-split_commits = false
-# regex for parsing and grouping commits
+[release]
+# Regex to select git tags that represent releases.
+# Example: "v[0-9].*"
+tags_pattern = "v[0-9].*"
+# Regex to select git tags that do not represent proper releases. Takes precedence over `release.tags_pattern`.
+# Changes belonging to these releases will be included in the next non-skipped release.
+# Example: "rc"
+skip_tags_pattern = ""
+# Whether to order releases chronologically or topologically.
+# Must be either `time` or `topology`.
+order_by = "time"
```

Comment:
should we merge `order_by` and `sort_order`?
E.g. introduce a new `sort` config that can be one of the following:
- newest
- oldest
- topology

Reply:
AFAIU `sort_order` describes sorting in commit groups, while `order_by` is for releases, so we can't merge unrelated options